pyomo/contrib/doe/doe.py

```diff
@@ -715,26 +540,25 @@ def _direct_kaug(self):
         measurement_index = []
         # produce the sensitivity for fixed variables
         zero_sens = np.zeros(len(self.param))
+        print(col)
 
         # loop over measurement variables and their time points
-        for measurement_name in self.measure.model_measure_name:
-            # get right line number in kaug results
-            if self.discretize_model:
-                # for DAE model, some variables are fixed
-                try:
-                    kaug_no = col.index(measurement_name)
-                    measurement_index.append(kaug_no)
-                    # get right line of dsdp
-                    dsdp_extract.append(dsdp_array[kaug_no])
-                except:
-                    self.logger.debug('The variable is fixed:  %s', measurement_name)
-                    # for fixed variables, the sensitivity are a zero vector
-                    dsdp_extract.append(zero_sens)
-            else:
-                kaug_no = col.index(measurement_name)
+        for mname in self.measure_name:
```

Comment:
`for i, mname in enumerate(self.measure_name)`? Or do I not understand this code? What is going on with i?

Reply:
I used these to change `C["CA", t]` to `C[CA, t]`, removing the "" outside a string. Now that we use the new way of generating measurements, this code is no longer needed. Removed.